README.md

```diff
@@ -45,7 +45,8 @@ The script must be run as root to do its magic but it will use a dedicated accou
   * Use `obfs4proxy-openvpn --export-cert -` on the server to get the required obfs4 *CERT* for the client.
   * [openvpn_client.conf.obfs4.sample](examples/openvpn_client.conf.obfs4.sample) / [openvpn_server.conf.obfs4.sample](examples/openvpn_server.conf.obfs4.sample) contain samples of OpenVPN client/server configurations.
 * [obfs4proxy-openvpn.service.sample](examples/obfs4proxy-openvpn.service.sample) contains sample of a systemd unit for obfs4proxy-openvpn.
-
+* Also in order to run the this script, pre-shared key must be created and also be imported to the client:
```

Comment:
Please also note that this is the case only if the user decides to use the default included openvpn conf files. The user can simply decide to go with a PKI implementation instead which wouldn't require a sharing pre-shared key.

Reply:
True

Reply:
No I meant that please change the wordings to reflect that. I don't think i can do this myself unless I push your changes and commit it again myself.
Though I'm not an expert in git